core-services/image-mirroring/konveyor/mapping_konveyor_latest

```diff
@@ -10,6 +10,8 @@ registry.ci.openshift.org/konveyor/velero-plugin-for-aws:velero-plugin-for-aws q
 registry.ci.openshift.org/konveyor/velero-plugin-for-aws:velero-plugin-for-aws-oadp-1.0 quay.io/konveyor/velero-plugin-for-aws:oadp-1.0-latest quay.io/konveyor/velero-plugin-for-aws:oadp-1.0
 registry.ci.openshift.org/konveyor/velero-plugin-for-aws:velero-plugin-for-aws-oadp-1.3 quay.io/konveyor/velero-plugin-for-aws:oadp-1.3-latest quay.io/konveyor/velero-plugin-for-aws:oadp-1.3
 registry.ci.openshift.org/konveyor/velero-plugin-for-aws:velero-plugin-for-aws-oadp-1.4 quay.io/konveyor/velero-plugin-for-aws:oadp-1.4-latest quay.io/konveyor/velero-plugin-for-aws:oadp-1.4
+registry.ci.openshift.org/konveyor/velero-plugin-for-legacy-aws:velero-plugin-for-legacy-aws quay.io/konveyor/velero-plugin-for-legacy-aws:latest
```

Comment:
/hold.. quay repo does not exists?

Reply:
https://quay.io/konveyor/velero-plugin-for-legacy-aws 404

Reply:
if this pr merged without quay repo, the entire mirroring job will fail, so it will affect other images.